Userland/Applications/Presenter/Presentation.cpp

```diff
@@ -42,6 +43,28 @@ StringView Presentation::author() const
     return "Unknown Author"sv;
 }
 
+Color Presentation::background_color() const
+{
+    if (m_metadata.contains("background-color"sv)) {
+        auto color = Color::from_string(m_metadata.get("background-color"sv)->view());
+        if (color.has_value()) {
+            return color.value();
+        }
+    }
+    return Color::White;
+}
+
+Color Presentation::foreground_color() const
+{
+    if (m_metadata.contains("foreground-color"sv)) {
+        auto color = Color::from_string(m_metadata.get("foreground-color"sv)->view());
+        if (color.has_value()) {
+            return color.value();
+        }
+    }
+    return Color::DarkGray;
+}
```

Comment:
Because we need them regularly, they can be extracted and defaulted at creation time. This kind of extraction is only done for less important data, such as the modification date.